### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by rastaman111** [zhiyongzou/iRead#3]
hello, I decided to add part of your project to my own and there was a problem with opening IRReaderCenterViewController, when I open it, a bunch of errors constantly appear

**Comment by rastaman111** [comment]
I sort of figured it out, but I don't understand why the very first picture is not loaded and if you go to the next page and return to 1, it crashes due to the fact that this page is not there

**Comment by zhiyongzou** [comment]
Can you provide more details? such as the crash log and the book

**Comment by rastaman111** [comment]
at first there was a problem with the pods 

def local_pods
  pod 'DTCoreText', :path => './DTCoreText'
  pod 'DTFoundation', :path => './DTFoundation'
end

when I installed them, an error occurs in the second pod that he does not see ![image](https://user-images.githubusercontent.com/15727518/100178889-ef2ca100-2ee5-11eb-85ec-691316d80bec.png) and I decided to install this pod not as a developer, but as a normal pod, I think there is a problem 

pod 'DTFoundation'

**Comment by zhiyongzou** [comment]
> at first there was a problem with the pods
> 
> def local_pods
> pod 'DTCoreText', :path => './DTCoreText'
> pod 'DTFoundation', :path => './DTFoundation'
> end
> 
> when I installed them, an error occurs in the second pod that he does not see ![image](https://user-images.githubusercontent.com/15727518/100178889-ef2ca100-2ee5-11eb-85ec-691316d80bec.png) and I decided to install this pod not as a developer, but as a normal pod, I think there is a problem
> 
> pod 'DTFoundation'

Whether the files(DTCoreText/DTFoundation/Profile) are in the same directory? As shown below 
![Xnip2020-11-25_12-43-46](https://user-images.githubusercontent.com/18436091/100184352-4baab300-2f1c-11eb-8965-f62519c1c85c.jpg)

**Book cover failed  to load may be due to `image` tag**

```
// DTHTMLAttributedStringBuilder.m

- (void)parser:(DTHTMLParser *)parser didStartElement:(NSString *)elementName attributes:(NSDictionary *)attributeDict
{
    // 兼容 XML image 标签
    if ([elementName isEqualToString:@"image"]) {
        elementName = @"img";
        NSString *xlink = @"xlink:href";
        NSMutableDictionary *temp = [attributeDict mutableCopy];
        id srcValue = [attributeDict objectForKey:xlink];
        if (srcValue != nil) {
            [temp setObject:srcValue forKey:@"src"];
            [temp removeObjectForKey:xlink];
            attributeDict = temp.copy;
        } else {
            return;
        }
    }
    ...
}
```

**Comment by rastaman111** [comment]
<img width="1403" alt="Снимок экрана 2020-11-25 в 07 59 48" src="https://user-images.githubusercontent.com/15727518/100185168-3b7edd80-2ef4-11eb-8d16-915c1b655071.png">

**Comment by rastaman111** [comment]
Fixed on this:
//#import "DTWeakSupport.h"
#import <DTFoundation/DTWeakSupport.h>

But the books never get the correct pages, 1 page with the picture does not load, the same Harry Potter book is displayed differently in the same font
I have 249 pages, 193 in your application

**Comment by rastaman111** [comment]
I figured out what's the matter
It was necessary to create a UITabBarController and specify the necessary functions there :)

**Comment by zhiyongzou** [comment]
Yes, you may be have missed some initial configuration. 

```
func initAfterViewDidAppear() {
        if initOnceAfterViewDidAppear {
            return
        }
        initOnceAfterViewDidAppear = true
        self.updateReadPageSzie()
        IRReaderConfig.initReaderConfig()
    }
    
    func updateReadPageSzie() {
        
        var safeInsets = UIEdgeInsets.zero
        if #available(iOS 11.0, *) {
            safeInsets = self.view.safeAreaInsets
        }
        
        if safeInsets.bottom == 0 || safeInsets.top == 0 {
            safeInsets = UIEdgeInsets.init(top: 30, left: 0, bottom: 30, right: 0)
        }
        
        let width = self.view.width - IRReaderConfig.horizontalSpacing * 2
        let height = self.view.height - safeInsets.top - safeInsets.bottom - IRReaderConfig.pageIndexSpacing
        
        IRReaderConfig.pageSzie = CGSize.init(width: width, height: height)
    }
```

**Comment by rastaman111** [comment]
That's just the point that I did not miss these 2 functions I indicated in the main VC, and not in TBC
